test-plan/eventing/event-delivery-conformance.md

```diff
@@ -0,0 +1,979 @@
+# Event Delivery
+
+From: https://github.com/knative/specs/blob/main/specs/eventing/control-plane.md#event-delivery
+
+
+From the Spec:
+
+>> Once a Trigger or Subscription has decided to deliver an event, it MUST do the
+following:
+>>
+>> 1. Read the resolved URLs and delivery options from the object's `status`
+   fields.
+>>
+>> 1. Attempt delivery to the `status.subscriberUri` URL following the
+   [data plane contract](./data-plane.md).
```

Comment:
can you fix this link?

Reply:
fixed https://github.com/knative/specs/pull/82/commits/e5c6215f8136df1e42299bd03c01acc589ce4537